51.md

```diff
@@ -110,3 +110,60 @@ An event with kind `30000` is defined as a parameterized replaceable list event
 ### Categorized Bookmarks List
 
 An event with kind `30001` is defined as a parameterized replaceable list event for categorizing bookmarks. The 'd' parameter for this event holds the category name of the list. Any standardized tag can be included in a Categorized Bookmarks List.
+
+#### Curated articles (topics)
```

Comment:
#### Categorized Articles List

An event with kind `30002` is defined as a parameterized replaceable list event for categorizing articles. The 'd' parameter for this event holds the category name of the list. The tags included in these lists MUST follow the format of 'a' tags for parameterized replaceable events as defined in [NIP-33 - Parameterized Replaceable Events](33.md#referencing-and-tagging).

Additional tags that can be included as metadata of these lists are:
- `title` 
- `description` 
- `thumbnail` 

Example:
```json
{
    "kind": 30002,
    "pubkey": "8f014c5aafb425b2fa414a49eb9ab17269f155937db67a5babe7bc40f9bbae3b",
    "created_at": 1685011028,
    "content": "",
    "tags": [
        [
            "d",
            "Bitcoin Cultural Movement - English"
        ],
        [
            "title",
            "Bitcoin Cultural Movement"
        ],
        [
            "description",
            "The advent of Bitcoin has not only revolutionized the financial landscape but has also given rise to a new cultural movement centered around this groundbreaking cryptocurrency. In this era of digital innovation, the concept of curation has emerged as a powerful tool to navigate and make sense of the vast amount of information and ideas surrounding Bitcoin. This article delves into the topic of Bitcoin curation, exploring its purpose, key players involved, and presenting five curated articles that shed light on different aspects of this phenomenon."
        ],
        [
            "thumbnail",
            "https://daorayaki-fs-bucket.s3.ap-east-1.amazonaws.com/8f014c5aafb425b2fa414a49eb9ab17269f155937db67a5babe7bc40f9bbae3b/files/1685007975118-DAORAYAKIS3.png"
        ],
        [
            "a",
            "30023:8f014c5aafb425b2fa414a49eb9ab17269f155937db67a5babe7bc40f9bbae3b:aVcTJ7_S0U8QyB1e3bLPD",
            ""
        ],
        [
            "a",
            "30023:8f014c5aafb425b2fa414a49eb9ab17269f155937db67a5babe7bc40f9bbae3b:LlGOjtygN7i4mX0CbLpEo",
            ""
        ]
    ],
    ...other fields
}
```

Reply:
Looks good. 

Although arguably all list types could have the tags below and we could use the `k` tag to be `30023` to allow filtering by blog posts only. 

```json
        [
            "title",
            "Bitcoin Cultural Movement"
        ],
        [
            "description",
            "The advent of Bitcoin has not only revolutionized the financial landscape but has also given rise to a new cultural movement centered around this groundbreaking cryptocurrency. In this era of digital innovation, the concept of curation has emerged as a powerful tool to navigate and make sense of the vast amount of information and ideas surrounding Bitcoin. This article delves into the topic of Bitcoin curation, exploring its purpose, key players involved, and presenting five curated articles that shed light on different aspects of this phenomenon."
        ],
        [
            "thumbnail",
            "https://daorayaki-fs-bucket.s3.ap-east-1.amazonaws.com/8f014c5aafb425b2fa414a49eb9ab17269f155937db67a5babe7bc40f9bbae3b/files/1685007975118-DAORAYAKIS3.png"
        ],
```

Reply:
+1. I'm actually adding title and description to all lists in the new Listr rebuild.

Reply:
'k' tag is a good idea. I still prefer a dedicated kind. not for technical reasons, just think its more consistent and straightforward for comprehension. also makes filtering easier/cleaner. 

but I will not be the main user of this feature so its really up to those who are invested in this list type whether they want a new dedicated kind or to just use existing list kind and add a 'k' tag.

Reply:
L

Reply:
@mostafa-khaldi L